osquery/core/init.cpp

```diff
@@ -365,7 +365,10 @@ void Initializer::start() {
   }
 
   // Load the osquery config using the default/active config plugin.
-  Config::getInstance().load();
+  auto s = Config::getInstance().load();
+  if (!s.ok()) {
+    LOG(ERROR) << "Error reading config: " << s.toString();
```

Comment:
This is an amazing find, it's a shame we never noticed "not having an error log on invalid config". Too much reliance on the specific plugin's error log, for invalid JSON and the like.